code/02_create-figures.R

```diff
@@ -0,0 +1,245 @@
+#########################################################################
+# Name of file - 02_create-figures.R
+# Data release - End of Life Publication
+# Original Authors - Alice Byers
+# Orginal Date - September 2019
+#
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+#
+# Description - Create and save figures for summary and report.
+#
+# Approximate run time - xx minutes
+#########################################################################
+
+
+### 1 - Setup environment and load functions ----
+
+source(here::here("code", "00_setup-environment.R"))
+source(here::here("functions", "summarise_data.R"))
+
+
+### 2 - Read in basefile ----
+
+basefile <- read_rds(here("data", "basefiles", 
+                          glue("{pub_date}_base-file.rds")))
+
+
+### 3 - Figure 1 - Trend Bar Chart ----
+
+fig1 <- 
+  
+  summarise_data(basefile, trend = TRUE) %>%
+  select(-deaths, -comm) %>%
+  mutate(qom_comm = 100 - qom) %>%
```

Comment:
I will change the column name to "qom_hosp" as it refers to the percentage spent in hospital.